app/test_main.py

```diff
@@ -1,3 +1,41 @@
+import pytest
 from app.main import get_human_age
 
 # write your code here
+
+
+class TestGetHumanAge:
+    @pytest.mark.parametrize(
+        "cat_age, dog_age",
+        [
+            ("2", "4"),
+            ((2,), (4,)),
+            ({2}, {4}),
+            ([2], [4]),
+            (None, None)
+        ]
+    )
+    def test_correct_value_types_get_human_age(
+            self, cat_age: int, dog_age: int
+    ) -> None:
+        with pytest.raises(TypeError):
+            get_human_age(cat_age, dog_age)
+
+    @pytest.mark.parametrize(
+        "cat_age, dog_age, expected_value",
+        [
+            (0, 0, [0, 0]),
+            (14, 14, [0, 0]),
+            (15, 15, [1, 1]),
+            (23, 23, [1, 1]),
+            (24, 24, [2, 2]),
+            (27, 28, [2, 2]),
+            (28, 29, [3, 3]),
+            (100, 100, [21, 17]),
+            (-3, -3, [0, 0])
+        ]
```

Comment:
add descriptive IDs for each set of test cases. This makes it clear which test cases are being executed when you run the tests.